recipes/loos/COPYING

```diff
@@ -0,0 +1,621 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Manually packaging license file isn't necessary since it is in the Github archive